addons/mail/static/src/composer/navigable_list.js

```diff
@@ -59,44 +59,28 @@ export class NavigableList extends Component {
     }
 
     get show() {
-        return Boolean(this.state.open && (this.state.isLoading || this.state.options.length));
+        return Boolean(this.state.open && (this.props.isLoading || this.state.options.length));
     }
 
     open() {
-        if (this.state.isLoading) {
-            return;
-        }
-        this.load().then(() => {
-            this.state.open = true;
-            this.navigate("first");
-        });
+        this.state.open = true;
+        this.load();
+        this.navigate("first");
     }
 
     close() {
         this.state.open = false;
         this.state.activeOption = null;
     }
 
-    async load() {
+    load() {
         this.state.options = [];
         const makeOption = (opt) => {
             return Object.assign(Object.create(opt), {
                 id: this.state.options.length,
             });
         };
```

Comment:
looks overkill (`Object.create`, check on `state.options.length`, `Object.assign`). shouldn't we use `.forEach((opt, index) =>` instead?

Reply:
I don't know, this is the original code, and I don't understand why it's done like this. 
Better not touch it I guess

Reply:
BTW, it's good practice to use for-of loop instead of `.forEach()` the suffix notation gives the wrong impression this has no side-effects when it usually is. For-of loop shows more clearly there are side-effects